aiotg/bot.py

```diff
@@ -87,14 +88,15 @@ def _webhook_loop(self, webhook_url, loop):
         Starts aiohttp web server.
         """
         app = web.Application(loop=loop)
+        port = os.environ.get('PORT', urlparse(webhook_url).port)
 
         async def handle(request):
             update = await request.json()
             self._process_update(update)
             return web.Response()
 
         app.router.add_route('POST', urlparse(webhook_url).path, handle)
-        web.run_app(app, port=urlparse(webhook_url).port)
+        web.run_app(app, port=int(port))
```

Comment:
this is gonna break on int(None) (urlparse(webhook_url).port will be None in case port is not specified in url).

I suggest to remove int() casting from this line and change the port line above to following:
```
port = int(os.environ.get('PORT', 0)) or urlparse(webhook_url).port
```